_posts/2019-03-23-WassersteinGAN.markdown

```diff
@@ -0,0 +1,522 @@
+categories: generative-adversarial-networks
+author: james
+blurb: "The Wasserstein GAN (WGAN) is a GAN variant which uses the 1-Wasserstein distance, rather than the KL-Divergence, to measure the difference between the model and target distributions. This seemingly simple change has big consequences! Not only does WGAN train more easily but it also achieves very impressive results &#8212; generating some stunning images."
+feedback: true
+---
+
+A number of people need to be thanked for their parts in making this happen. Thank you to Martin Arjovsky, Avital Oliver, Cinjon Resnick, Marco Cuturi, Kumar Krishna Agrawal, and Ishaan Gulrajani for contributing to this guide. 
+
+Of course, thank you to Sasha Naidoo, Egor Lakomkin, Taliesin Beynon, Sebastian Bodenstein, Julia Rozanova, Charline Le Lan, Paul Cresswell, Timothy Reeder, and Michał Królikowski for beta-testing the guide and giving invaluable feedback. A special thank you to Martin Arjovsky, Tim Salimans, and Ishaan Gulrajani for joining us for the weekly meetings.
+
+Finally, thank you to Ulrich Paquet and Stephan Gouws for introducing me to Cinjon.
+
+<div class="deps-graph">
+  <iframe class="deps" src="/assets/wgan-deps.svg" width="400"></iframe>
+  <div>Concepts used in Wasserstein GAN. Click to navigate.</div>
+</div>
+
+# Why
+
+The Wasserstein GAN (WGAN) is a GAN variant which uses the 1-Wasserstein distance, rather than the KL-Divergence, to measure the difference between the model and target distributions. This seemingly simple change has big consequences! Not only does WGAN train more easily (a common struggle with GANs) but it also achieves very impressive results &#8212; generating some stunning images. By studying the WGAN, and its variant the WGAN-GP, we can learn a lot about GANs and generative models in general. After completing this curriculum you should have an intuitive grasp of why the WGAN and WGAN-GP work so well, as well as, a thorough understanding of the mathematical reasons for their success. You should be able to apply this knowledge to understanding cutting edge research into GANs and other generative models.
+
+<br />
+
+# 1 Basics of Probability & Information Theory
+  **Motivation**: To understand GAN training (and eventually WGAN & WGAN-GP) we need to first have some understanding of probability and information theory. In particular we will focus on Maximum Likelihood Estimation and the KL-Divergence. This week we will make sure that we understand the basics so that we can build upon them in the following weeks.  
+
+  _This week contains some fairly introductory material, if everyone participating in this curriculum is comfortable with this material you may wish to treat this week as optional or as a pre-requisite. However, this week still covers some very interesting and cool topics, and it is important to have a solid grasp of these concepts in order to build towards understanding the Wasserstein GAN, so skip with caution._
+
```

Comment:
Not super sure about this comment. It seemed like many people in the group found the first week a bit dull (too introductory) so I wanted to have some kind of warning that people might want to give it a skip. However, I think that a lot of the concepts from this week are not only very interesting but also important for the following weeks so I don't want to encourage too many people to skip it!

Reply:
You can feel free to keep this, but also keep in mind that more often than not, I imagine a large part of the target audience will be individuals doing this on a self-paced time. 

I'd totally support you removing this comment entirely if you wanted, but also I think it's fine if you want to keep it. What do you think?

Reply:
I'm still undecided about this. I am leaning towards just removing it. For now, I've left it in though. I think that if most people are doing this curriculum in a self-paced setting, they can pretty quickly figure out which material is too introductory for them and then just skip it - we don't need to tell them :)

Reply:
I decided to remove this comment.